modules/mobile/docs/openapi.yaml

```diff
@@ -1922,6 +1922,18 @@ paths:
           required: true
           schema:
             type: object
+        - description: The page number requested
```

Comment:
Since you're removing pagination, I think you need to remove this as well.

Reply:
good catch. fixed